third_party/silabs/SiWx917_sdk.gni

```diff
@@ -104,7 +104,7 @@ template("siwx917_sdk") {
       "${efr32_sdk_root}/util/third_party/freertos/kernel/include",
       "${wifi_sdk_root}/components/protocol/wifi/si91x",
       "${efr32_sdk_root}/util/third_party/freertos/kernel/portable/GCC/ARM_CM4F",
-      "${examples_plat_dir}/device/inc",
+      "${efr32_sdk_root}/matter/si91x/siwx917/BRD4325x/support/inc/inc",
```

Comment:
typo, should be the matter_support path here.

Reply:
done